managers/data_context.go

```diff
@@ -14,30 +17,19 @@ import (
 // DataContext is a common interface to implement operations with authorization server entities (data.Realm, data.Client, data.User)
 // now contains only set of Get methods, during implementation admin CLI should be expanded to create && update entities
 type DataContext interface {
-	// GetRealm returns realm by name (unique)
-	GetRealm(realmName string) *data.Realm
+	// GetRealmWithClients returns realm by name (unique) with all clients
+	GetRealmWithClients(realmName string) (*data.Realm, error)
```

Comment:
I think we should nor rename this method because we don't return a Realm without Clients, we should add a commentaries about behaviour